library/init/meta/expr.lean

```diff
@@ -282,18 +284,8 @@ meta def binding_body : expr → expr
 | (lam _ _ _ b) := b
 | e             := e
 
-meta def prop : expr := expr.sort level.zero
-
 meta def imp (a b : expr) : expr :=
-pi `a binder_info.default a b
-
-meta def and_ (a b : expr) : expr :=
-app (app (const ``and []) a) b
-
-meta def not_ (a : expr) : expr :=
-app (const ``not []) a
-
-meta def false_ : expr := const ``false []
```

Comment:
I'm glad to get rid of most of these functions, but `imp` turned out to be just too convenient.